.travis.yml

```diff
@@ -62,11 +62,15 @@ matrix:
     # Specify SQLALCHEMY=latest to avoid errors installing.
     - python: "2.6"
       env: TWISTED=14.0.2 TESTS=trial_worker SQLALCHEMY=latest
-    - python: "2.6"
```

Comment:
its tested by bbtravis anyway. As travis is slower, I think we need to remove as much as possible stuff from travis